CHANGELOG.md

```diff
@@ -1,5 +1,111 @@
 # Releases
 
+## Version 0.14.0
+
+Date: 2022-09-30
+
+This release focuses on two main themes, support for running Panel entirely in the browser using WebAssembly (via Pyodide and PyScript), quality of life improvements to make it easier to build responsive and performant applications and finally improvements to static typing and docstrings to enhance the developer experience.
```

Comment:
```suggestion
This release focuses on three main themes, support for running Panel entirely in the browser using WebAssembly (via Pyodide and PyScript), quality of life improvements to make it easier to build responsive and performant applications and finally improvements to static typing and docstrings to enhance the developer experience.
```

Isn't this three things?
1) support for running Panel entirely in the browser using WebAssembly (via Pyodide and PyScript
2) quality of life improvements to make it easier to build responsive and performant applications
3) improvements to static typing and docstrings to enhance the developer experience